src/connectors/configure-related-items/connectConfigureRelatedItems.ts

```diff
@@ -148,11 +133,8 @@ See https://www.algolia.com/doc/api-reference/api-parameters/optionalFilters/
     const makeConfigure = connectConfigure(renderFn, unmountFn);
 
     return {
-      ...makeConfigure({ searchParameters }),
-
+      ...makeConfigure({ searchParameters } as any),
```

Comment:
Do you mind explaining why casting as `any` is required here? There should be a way to avoid this.

Reply:
This is not a easy one. I may need to explain you an zoom, and write here the feedback later.

Reply:
I'm also surprised here, I assume that a configure widget should accept searchParams

Reply:
fixed.